### PR TITLE
LPS-144039 Update 5% discount shown at payment method

### PR DESCRIPTION
**Pull request by samuelleolopes**

### CODE REVIEW

**Comment by samuelleolopes** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html><details><summary>More details about: <b>LPS-144039 Update 5% discount shown at payment method</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-144039">Update 5% discount shown at payment method</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Technical Task</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Média</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>Aberto</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Samuel Lopes</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Ilza Medeiros</td>
        </tr>
                </table>
          </details></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

1 file changed, 4 insertions(+), 2 deletions(-)

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ const PaymentMethod = () => {
 	const [agree, setAgree] = useState(false);
 	const [{orderId, product}] = useContext(SelectedQuoteContext);
 	const [methods, setMethods] = useState([]);
+	const productDiscounted = product.price * 0.95;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:15]: Hi @samuelleolopes 

I'm not sure, but I think this product.price is coming as a string.

And you are multiplying then. Something like '390' * 0.95

Javascript still evaluate the value, but it not trustful

```suggestion
	const productDiscounted = Number(product.price) * 0.95;
```

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js [hunk 2]
```diff
@@ -12,6 +12,8 @@ const PaymentMethod = () => {
...
+	const productPromo = product.price * 0.05;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:16]: Same from the other

```suggestion
	const productPromo = Number(product.price) * 0.05;
```

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js [hunk 3]
```diff
@@ -35,11 +37,11 @@ const PaymentMethod = () => {
 						{
 							checked: true,
 							description: `Save $${Number(
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:39]:
```suggestion
							description: `Save $${
```

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js [hunk 4]
```diff
@@ -35,11 +37,11 @@ const PaymentMethod = () => {
...
-								product.promo
+								productPromo
 							).toLocaleString('en-US')}`,
 							id: 0,
 							title: `Pay in full – $${Number(
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:43]:
```suggestion
							title: `Pay in full – $${
Write

```

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js (hunk 3 above)
Comment (kevenleone) [reply modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:39]: Remove Number from here;

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js (hunk 4 above)
Comment (kevenleone) [reply modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:43]: Remove Number from here;

**Review by kevenleone — CHANGES_REQUESTED** [review]
It's looks good, but look at my changes @samuelleolopes

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js (hunk 4 above)
Comment (samuelleolopes) [reply modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:43]: Done

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js (hunk 3 above)
Comment (samuelleolopes) [reply modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:39]: Done

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js (hunk 1 above)
Comment (samuelleolopes) [reply modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:15]: Done

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js (hunk 2 above)
Comment (samuelleolopes) [reply modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/PaymentMethod/index.js:16]: Done

**Comment by kevenleone** [comment]
ci:forward

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**